client/src/templates/Challenges/rechallenge/transformers.js

```diff
@@ -213,6 +214,35 @@ const transformHtml = async function (file) {
   return transformContents(() => div.innerHTML, file);
 };
 
+// Find if the base html refers to the css or js files and record if they do. If
+// the link or script exists we remove those elements since those files don't
+// exist on the site, only in the editor
+const transformIncludes = async function (fileP) {
+  const file = await fileP;
+  const div = document.createElement('div');
+  div.innerHTML = file.contents;
+  const link =
+    div.querySelector('link[href="styles.css"]') ??
+    div.querySelector('link[href="./styles.css"]');
+  const script =
+    div.querySelector('script[src="script.js"]') ??
+    div.querySelector('script[src="./script.js"]');
+  const importedFiles = [];
+  if (link) {
+    importedFiles.push('index.css');
+    link.remove();
+  }
+  if (script) {
+    importedFiles.push('index.js');
+    script.remove();
```

Comment:
Removing these means I won't be able to test for them in the document.querySelector results, right?

Reply:
Right. They were causing 404 errors to appear in the console, so I figured it was simpler just to remove them entirely.

Reply:
Having said that, if you need something in a test, we can always add attributes and so on.  You want to keep, say, an empty `<script>` element with a custom `data-fcc="js"` attribute?  Something like that?

Reply:
Nah, I was actually testing that the link element to link the CSS existed lol

Reply:
Okay, cool. If you can manage without, let's keep it simple and keep the number of things in the frame to a minimum.